blackbox/docs/src/crate/tests.py

```diff
@@ -92,7 +109,31 @@ def tearDown(self):
         super(ConnectingCrateLayer, self).tearDown()
 
 
-empty_layer = ConnectingCrateLayer(
+class JavaReplLayer(object):
+
+    __bases__ = ()
+    __name__ = 'javarepl'
```

Comment:
I think it would make sense to move those two attributes into the JavaRepl class and then use JavaRepl as layer directly, without wrapper.